### DISCUSSION TITLE
Is fun bot mod offline

### DISCUSSION THREAD

**Discussion by Mohammadmnmx** [Joe91/fun-bots#291] [Q&A] [answered]
Do still i need interrnet to play the mod?

**Comment by Firjens** [comment, marked as answer]
This mod only connects to the internet to check if an update is available. You can play fun-bots without an internet connection. However Venice Unleashed's offline support is limited, you should contact the authors of Venice Unleashed to know more about the ability to play offline.

**Reply by Mohammadmnmx** [reply]
Thank you

**Reply by Mohammadmnmx** [reply]
Anothet question
Whene i go to download page in this website there is 2 zip file there names source.zip
So i download the both files or just one

**Reply by Firjens** [reply]
You should get the latest release from the release page and not download the source code directly from the repository.

**Comment by Joe91** [comment]
As far as I know it's not possible to start VU without internet.

**Reply by Mohammadmnmx** [reply]
Ya i know but that was an old virsion
Maybe now after updates we can play it offline

**Reply by BaumTV** [reply]
We need someone to port his mod to BF - Portable. It cannot be that VU gets all the love and after it shuts down, we all are screwed. Be independent guys.
